src/ResourceManager.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <unordered_map> //HashMap
+#include <SFML/Graphics.hpp>
+#include <SFML/Audio.hpp>
+
+
+class ResourceManager;
+
+class ResourceManager {
+private:
+	ResourceManager(); //constructor
+	void LoadData();
+
+public:
+	ResourceManager(const ResourceManager&) = delete; // copy constructor cannot be used
+	void operator = (const ResourceManager&) = delete;  //assignmnent is locked
+	~ResourceManager();  // destructor
+
+	static ResourceManager& Get() {
+		static ResourceManager instance;
+		return instance;
+	}
+
+	sf::Texture GetTexture(const char);
+	sf::Font GetFont(const char);
+	sf::Sound GetSound(const char);
+
+private:
+	sf::Texture duckyTexture;
+
+
+	std::unordered_map<const char, sf::Texture> TextureMap = {
```

Comment:
no need to initailize it here :) it should be done by the LoadResources function